roles/swift-proxy/tasks/main.yml

```diff
@@ -1,17 +1,18 @@
 ---
 - name: permit swift proxy from anywhere (ubuntu)
```

Comment:
delete setting firewall rules for ufw and firewall from here and instead add call in meta to openstack-firewall 

example: https://github.com/blueboxgroup/ursula/blob/master/roles/neutron-control/meta/main.yml#L9